tests/test_other.py

```diff
@@ -1,9 +1,14 @@
 # Standard Python modules
+import os
 import sys
 import unittest
 
+# we have to unset this environment variable because otherwise when we import `_import_snopt_from_path`
+# the snopt module gets automatically imported, thus failing the import test below
+os.environ.pop("PYOPTSPARSE_IMPORT_SNOPT_FROM", None)
```

Comment:
Was this working for the CI because we don't set the environment variable in the Docker images?

Reply:
Correct. That being said, the test still fails unless it is run with the `isolated` flag from `testflo`, because it's possible for a previous test to import `SNOPT` first, thereby importing the `snopt` module. I'm not sure of an easy way to fix this if the behaviour we want is to import the `snopt` module if possible at the start of the pySNOPT module.

Reply:
This does not impact CI but will impact people locally who are using this env var.